tests/Fixer/Import/GroupImportFixerTest.php

```diff
@@ -291,25 +306,41 @@ public function provideFixCases()
 use Framework\Baz\Class6;
 use Framework\Foo\Class7;
 ',
-            ],
-            [
-                '<?php
+        ];
+
+        yield [
+            '<?php
 
 use function Foo\baz;
 use Foo\{Bar, Baz};
 ',
-                '<?php
+            '<?php
 
 use Foo\Bar;
 use function Foo\baz;
 use Foo\Baz;
 ',
-            ],
         ];
 
-        foreach ($tests as $index => $test) {
-            yield $index => $test;
-        }
+        yield [
```

Comment:
Sorry for the noise, I replaced the array with `yield` statements. The failing case is added here.

Reply:
not only noise, but also conflicts... :(